compiler/src/iree/compiler/Dialect/Stream/Transforms/PackDispatchOperands.cpp

```diff
@@ -26,6 +26,17 @@ namespace IREE {
 namespace Stream {
 namespace {
 
+//===----------------------------------------------------------------------===//
+// Bitwidth utilities
+//===----------------------------------------------------------------------===//
+static int64_t getTypeBitwidth(Type type) {
```

Comment:
style: vertical whitespace
```suggestion

static int64_t getTypeBitwidth(Type type) {
```